service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/FailedPBAPaymentEventHandler.java

```diff
@@ -27,32 +36,72 @@
     private final FailedPBAPaymentContentProvider notificationContent;
 
     @EventListener
-    public void notifyLocalAuthority(FailedPBAPaymentEvent event) {
+    public void notifyApplicant(FailedPBAPaymentEvent event) {
+        CaseData caseData = event.getCaseData();
+        String applicant = defaultIfNull(event.getApplicantName(), "");
+        if (event.getApplicationTypes().contains(ApplicationType.C110A_APPLICATION)) {
+            notifyLocalAuthority(event, APPLICATION_PBA_PAYMENT_FAILED_TEMPLATE_FOR_LA);
+        } else {
+            if ((caseData.getCaseLocalAuthorityName() + ", Applicant").equals(applicant)) {
```

Comment:
that looks a bit dodgy, this component should not need to know how things are formatted in other component.
Maybe an option is to use ApplicantsListGenerator to get applicant name and type. Ideally event should pass not formatted string like "John Smith, Respondent 1", but rather some compound object like applicant : {
type: LA, RESPONDENT,OTHER,
name: ""
}